### ISSUE TITLE
excel and csv's 

### ISSUE THREAD

**Issue by pepelisu** [rethomics/damr#9]
If a csv file is open with excel it automatically converts the datatime columns to data format, this will convert this columns into a number when is saved to csv again or in a format that is not working in the import. 
This is not a bug, is just a heads up. This should be explained in the documentation. 
If xlsx file is open and modified and saved like csv there is no problem, as far as the datetime columns are formatted as "text".

**Comment by qgeissmann** [comment]
yes, it is a bit of a pain (see https://stackoverflow.com/questions/165042/stop-excel-from-automatically-converting-certain-text-values-to-dates). Will think about the best way around for general public...

**Comment by DenaFrancis** [comment]
I am having a similar issue with your tutorial. Everytime I try to link the dams data with the metadata file I get this error:
> metadata <- link_dam_metadata(metadata, result_dir = DATA_DIR)
Warning: 1 parsing failure.
row col   expected               actual
  1  -- date like  1/7/2017  8:00:00 am

Error: 1 parsing failure

**Comment by qgeissmann** [comment]
yes, maybe we should come up with a way to load excel files instead of csv (and fix excel tendency to try to be cleverer than the user...). for now, can you make it work by forcing the date column to be a string?

**Comment by zoelang30** [comment]
Has this been solved or is there a work around?

I am getting the same error as @DenaFrancis 
Warning: 1 parsing failure.
row col   expected                      actual
  1  -- date like  "=""04/07/2022" ="15:30:00"

Error: 1 parsing failure


Any help on this please :)
